vlayer/ethereum/circuits/lib/src/README.md

```diff
@@ -0,0 +1,49 @@
+# Slots
+
+This is a [Noir](https://noir-lang.org) library for slots.
```

Comment:
I'd add just few words explanation what slots are we talking about here

Reply:
In the next PR I will add a whole file with explanation about Solidity Storage Layout and it will be linked here.